src/tools/compiletest/src/runtest.rs

```diff
@@ -162,20 +162,7 @@ pub fn run(config: Config, testpaths: &TestPaths) {
     };
     base_cx.init_all();
 
-    if base_props.revisions.is_empty() {
-        base_cx.run_revision()
-    } else {
-        for revision in &base_props.revisions {
-            let revision_props = TestProps::from_file(&testpaths.file, Some(revision), &config);
-            let rev_cx = TestCx {
-                config: &config,
-                props: &revision_props,
-                testpaths,
-                revision: Some(revision),
-            };
-            rev_cx.run_revision();
-        }
-    }
+    base_cx.run_revision();
```

Comment:
....so that here we can be given the revision, and load up the `revision_props` and so forth like we used to do